core/services/ocr2/plugins/ccip/testhelpers/integration/jobspec.go

```diff
@@ -62,6 +70,7 @@ func (o *OCR2TaskJobSpec) String() (string, error) {
 	}{
 		Name:                  o.Name,
 		JobType:               o.JobType,
+		ExternalJobID:         externalID,
```

Comment:
This is required for JD to propose the job, without ExternalJobID it's throwing error